docs/source/iam-group.rst

```diff
@@ -11,18 +11,26 @@ modify, and destroy all of the BinaryAlert infrastructure:
 
     # ========== Variables ==========
 
-    variable "account" {
-      default = "123412341234"  # Replace with your account ID
-    }
+    variable "aws_access_key" {}
+    variable "aws_secret_key" {}
```

Comment:
Sorry the documentation was confusing!

AWS access keys should never be set with terraform variables directly - this runs the danger of accidentally checking them into variable files or typing them on the command line where security tools like osquery could pick them up.

You should be able to use [environment variables](https://www.terraform.io/docs/providers/aws/#environment-variables) instead, feel free to update the documentation to this effect.

You said you tried environment variables and it still didn't work? What are the names of the variables you ahve in `env | grep AWS`

Reply:
No worries! I see what happened here.

When I first started working on this I tried getting this Terraform file to work with both a Git-ignored `terraform.tfvars` file and `TF_VAR_…` environment variables so I could avoid typing my credentials in on the command line for the reasons you pointed out. Neither worked without explicitly defining the AWS Provider which is what I opened this pull request to address.

I hear your concerns about defining these variables. I didn't realize the AWS provider was automatically looking for those `AWS…` environment variables as I don't keep them set by default in my development environment. This Terraform file is working for me now with them set.

I dropped these commits in favor of a minor documentation update noting that.